.github/workflows/continous-integration-os.yml

```diff
@@ -61,9 +61,9 @@ jobs:
   test:
     name: Test ${{ matrix.rust }} on ${{ matrix.os }}${{ matrix.features }}
     # The large timeout is to accommodate:
-    # - Windows builds (75 minutes, typically 30-50 minutes), and
-    # - parameter downloads (an extra 90 minutes, but only when the cache expires)
-    timeout-minutes: 165
+    # - macOS and Windows builds (90 minutes, typically 30-70 minutes), and
+    # - parameter downloads (an extra 100 minutes, but only when the cache expires)
+    timeout-minutes: 190
```

Comment:
Why not the default 6hrs?

Reply:
Because if our builds suddenly jump from 3 to 6 hours, we likely have a compile or production performance bug, and we shouldn't merge that PR.

We don't do any other performance monitoring, so I don't want to increase these timeouts too much.